src/grpc/config/hpr_route.erl

```diff
@@ -97,7 +100,13 @@ lns(Route) ->
     Server = ?MODULE:server(Route),
     Host = ?MODULE:host(Server),
     Port = ?MODULE:port(Server),
-    <<Host/binary, $:, (erlang:integer_to_binary(Port))/binary>>.
+    case Server#config_server_v1_pb.protocol of
+        {http_roaming, _RoamingProtocol} ->
+            Path = http_roaming_path(Route),
+            <<Host/binary, $:, (erlang:integer_to_binary(Port))/binary, Path/binary>>;
```

Comment:
@barndon note we should make sure `/` is present if a path is filled out.

Reply:
There is an Erlang function that take care of that (https://www.erlang.org/doc/man/uri_string.html#recompose-1) ex:
```
uri_string:recompose(#{host => "localhost", path => "test/123", scheme => "http"}).
"http://localhost/test/123"
```